tools/package-tools/build-bundles.ts

```diff
@@ -185,8 +190,18 @@ export class PackageBundler {
       return map;
     }, {} as {[key: string]: string});
   }
-}
 
+  /** Gets the AMD module name for a package and an optional entry point. */
+  private getAmdModuleName(packageName: string, entryPointName?: string) {
+    let amdModuleName = `ng.${dashCaseToCamelCase(packageName)}`;
```

Comment:
Can you add a comment that shows some of the example expected amd module names from the respective entry points?